tools/python/paladin_claiming/recycle_bribes.py

```diff
@@ -0,0 +1,195 @@
+import json
+from dotenv import load_dotenv
+from typing import List
+import sys
+from pathlib import Path
+import os
+from datetime import datetime
+
+sys.path.append(str(Path(__file__).parent.parent))
+
+import requests
+from web3 import Web3
+from helpers.hh_bribs import get_hh_aura_target
+from bal_addresses import AddrBook
+from bal_tools.safe_tx_builder import SafeTxBuilder, SafeContract
+from bal_tools import Web3RpcByChain
+
+
+load_dotenv()
+
+SCRIPT_DIR = Path(__file__).parent
+CLAIM_OUTPUT_DIR = SCRIPT_DIR / "claim_output"
+PAYLOAD_DIR = SCRIPT_DIR / "payload"
+
+CLAIM_OUTPUT_DIR.mkdir(parents=True, exist_ok=True)
+PAYLOAD_DIR.mkdir(parents=True, exist_ok=True)
+
+CURRENT_DATE = datetime.utcnow().strftime("%Y-%m-%d")
+
+w3_by_chain = Web3RpcByChain(os.getenv("DRPC_KEY"))
```

Comment:
missing `DRPC_KEY=` in `tools/python/.env.template`

Reply:
added in b77dd9f